app/pulp/app/settings.py

```diff
@@ -165,18 +165,22 @@
             'simple': {'format': 'pulp: %(name)s:%(levelname)s: %(message)s'},
         },
         'handlers': {
-            'stream': {
+            'pulp': {
```

Comment:
Can this be named the 'stream' handler since it is actually `logging.StreamHandler`?